src/note.cpp

```diff
@@ -879,7 +879,7 @@ int Note::CalcLedgerLines(FunctorParams *functorParams)
 
     // HARDCODED
     const int legerLineExtension
```

Comment:
Sorry to be picky, but you missed that one...

Reply:
No, that's my bad. Fixing now.